tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -106,22 +105,22 @@ public enum CompressionType
             NormalData
         }
 
-        [Benchmark]
+        [Benchmark(InnerIterationCount = 10000)]
```

Comment:
Use a constant to set InnerIterationCount, so it is easier to change (will only need to change in one place).